2021/scipy/README.md

```diff
@@ -0,0 +1,44 @@
+# SciPy 2021 Tutorial - _Fairness in AI Systems: From Social Context to Practice using Fairlearn_
+
+Tuesday, July 13 from 9:00 AM - 1:00 PM CDT
+
+---
+
+Fairness in AI systems is an interdisciplinary field of research and practice that aims to understand and address some of the negative impacts of AI systems on society. In this tutorial, we will walk through the process of assessing and mitigating fairness-related harms in the context of the U.S. health care system. This tutorial will consist of a mix of instructional content and hands-on demonstrations using Jupyter notebooks. Participants will use the Fairlearn library to assess an ML model for performance disparities across different racial groups and mitigate those disparities using a variety of algorithmic techniques.
+
+## Set-up Instructions
+
+---
+
+### Option 1: Local Set-Up
+
+If you want to follow along in this tutorial on your local machine, we recommend using the [Anaconda Python distribution](https://www.anaconda.com/products/individual).
+
+For this tutorial, participants will need to download the Jupyter notebook _name goes here_.
+
+In a Python virtual environment (Python version >= 3.7), install the necessary libraries by running
+
+```
+pip install -r requirements.txt
+```
+
+
+### Option 2: Google Colab
```

Comment:
Colab should be the main option for this, local set-up the secondary option.

Reply:
(but i don't care too strongly)

Reply:
Binder?